packages/config/README.md

```diff
@@ -2,6 +2,74 @@
 
 ## Usage
 
+### `.bemrc` fields
+
+Field   | Type    | Purpose
+---     | ---     | ---
+root    | `Boolean` | Used to determine the root directory. Configs in parent dirs won't be gathered
+naming  | `string`, `Object` | Name of one of [naming presets](https://github.com/bem/bem-sdk/tree/master/packages/naming.presets) or custom naming definition
+levels  | `Array<LevelConf>` | List of known levels in the right order<br> (usually local) with their configurations
```

Comment:
Feels like we don't need the difference between levels and libraries — in all cases they are the same.

Let's drop this field in favor of `layerOrder: string[]` since we need to only to define ordering between layers for now and it's almost useless logic for the most of our users.

If we need to configure some directory (originally levels) with additional fields we should convert them to libraries and define in `libs` field. `path` there would start with `'./'` as it does in nodejs's `require` function so it will be very familiar too.

So:
- Dropping `levels`
- Adding `layerOrder`
- Filtering libraries, if need, to get local ones (configured "levels")